### PR TITLE
Refactor the ONIE documentation

### PR DESCRIPTION
**Pull request by whowutwut**
I think some confusion when we started asking some folks to review.  So I am going through an End to End review of the documentation and trying to clarify the onie support 

For review, you can see.. 
http://c910loginx03.pok.stglabs.ibm.com/~vhu/onie/advanced/networks/index.html

### CODE REVIEW

**Comment by whowutwut** [comment]
@immarvin  @cxhong  The pull request is ready for review and merge.

File: docs/source/advanced/networks/onie_switches/os_cumulus/install.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+#. Set the ``provmethod`` of the target switch to the Cumulus Linux install image:  ::
+
+    chdef <switch> provmethod="/install/custom/sw_os/cumulus/cumulus-linux-3.1.0-bcm-armel.bin"
+
+#. Run ``makedhcp`` to prepare the DHCP/BOOTP lease information for the switch: ::
+
+    makedhcp -a <switch> 
+
+
+At this point, the DHCPREQUEST from the switch should now get a response with the Cumulus Linux OS and begin the network installation.  *(Normal  installation time for Cumulus Linux is 1 hour)*
```
Comment (immarvin) [inline docs/source/advanced/networks/onie_switches/os_cumulus/install.rst]: what to do at this point to start provision? should better give some instruction

Comment (whowutwut) [inline docs/source/advanced/networks/onie_switches/os_cumulus/install.rst]: I agree,... it's quite confusing...    I am running the steps from scratch today and it seems like once we do the `makdhcp -a <switch>`, the dhcp is still running on the switch and the response will now start sending the file....  

we can see some logs in `/var/log/httpd/access_log` and also monitor the activity in `/var/log/xcat/computes.log`...   

@immarvin @cxhong any suggestions on how to improve this?

Comment (immarvin) [inline docs/source/advanced/networks/onie_switches/os_cumulus/install.rst]: hi @whowutwut , "the dhcp is still running",  you mean you can see the switch is keep on sending DHCPDISCOVER? this is the designed behavior of ONIE switch, if there is no cumulus OS installed or you issued "once-select -i;reboot" to issue a reinstall, the switch will keep on sending DHCPDISCOVER once it's powered on.

how about issue the reinstall or reset the switch power after "makedhcp -a"?

**Comment by whowutwut** [comment]
Running through the documentation, I see the following messages come out when we run the `configinterface` scripts the first time... 

```
[root@fs3 postscripts]# updatenode frame1sw1 -P configinterface.vhu
frame1sw1: xcatdsklspost: downloaded postscripts successfully
frame1sw1: /tmp/fileADSxpq.dsh: line 66: msgutil_r: command not found
frame1sw1: Wed Feb  8 19:37:20 UTC 2017 Running postscript: configinterface.vhu
frame1sw1: postscript: configinterface.vhu exited with code 0
frame1sw1: Running of postscripts has completed.
frame1sw1: /tmp/fileADSxpq.dsh: line 66: msgutil_r: command not found
``` 

Afterwards, it seems like we don't see the problem again... 

```
[root@fs3 postscripts]# time updatenode frame1sw1 -P configinterface.vhu
frame1sw1: xcatdsklspost: downloaded postscripts successfully
frame1sw1: Wed Feb  8 19:41:26 UTC 2017 Running postscript: configinterface.vhu
frame1sw1: postscript: configinterface.vhu exited with code 0
frame1sw1: Running of postscripts has completed.

real	2m26.395s
user	0m0.267s
sys	0m0.005s

```

This seems like part of the xcat postscript framework that some files are not there on the Cumulus Switch initially? 

The time that it takes to run these scripts is quite long...

**Comment by immarvin** [comment]
hi @whowutwut , the error message is a known issue, fixed in https://github.com/xcat2/xcat-core/issues/2473

**Comment by whowutwut** [comment]
@immarvin  with the last commit 8c5195b I've tried to explain how the install gets kicked off.  If you are ok with the changes, I think this documentation is ready to be merged.   

 I did notice that in the `/etc/dhcp/dhcpd.conf` file, xCAT is still creating entries for the onie install, I think that can be removed as that is not being used anymore. 
```
  } else if substring (option vendor-class-identifier,0,11) = "onie_vendor" { #for onie on cumulus switch
      option www-server = "http://50.3.27.1/install/onie/onie-installer";
```

If you agree, can you or @cxhong  remove it in another PR?

**Comment by immarvin** [comment]
hi @whowutwut ,  I will merge the PR.

on the line in ``/etc/dhcp/dhcpd.conf``,
````
    } else if substring (option vendor-class-identifier,0,11) = "onie_vendor" { #for onie on cumulus switch
      option www-server = "http://50.3.25.1/install/onie/onie-installer";
````
I think this line is for dynamic range and should be preserved, what do you think? @whowutwut @cxhong

**Comment by whowutwut** [comment]
Regarding that line. I don't think it's being used.. is it? /install/onie/onie-installer might have been the install file during some investigation effort. But I think it's now the prov method line that points to the image file. Then makedhcp will create an entry for the MAC address. @cxhong can you confirm?

**Pull request merged**